src/shared/HOC/body-disable-HOC.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
```

Comment:
A question about naming. Please use one of the next conventions:
1. with-body-disable
2. body-disable-decorator